src/NetAnalyzers/CSharp/Microsoft.CodeQuality.Analyzers/Maintainability/CSharpReviewUnusedParameters.Fixer.cs

```diff
@@ -1,28 +0,0 @@
-// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
-
-using System.Composition;
-using Microsoft.CodeAnalysis;
-using Microsoft.CodeAnalysis.CodeFixes;
-using Microsoft.CodeAnalysis.CSharp;
-using Microsoft.CodeQuality.Analyzers.Maintainability;
-
-namespace Microsoft.CodeQuality.CSharp.Analyzers.Maintainability
-{
-    /// <summary>
-    /// CA1801: Review unused parameters
-    /// </summary>
-    [ExportCodeFixProvider(LanguageNames.CSharp), Shared]
-    public sealed class CSharpReviewUnusedParametersFixer : ReviewUnusedParametersFixer
```

Comment:
IDE0060 does not have a code fix. CodeFix for CA1801 had some shortcomings and heuristics. I think we should consider opening a Roslyn issue to author a code fix for IDE0060, which uses this code fix implementation as a reference.